spec/features/categories/user_can_see_all_categories_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "when a user visits the categories show page" do
+  scenario "they see a list of categories" do
+
+  CATEGORIES = ["Technology", "Human Resources", "Administration"]
+  CATEGORIES.each do |category|
```

Comment:
feel free to do things like 

```
3.times do |i|
Categorie.create(title: "Category#{i}"
end
```

It's sometimes a little faster than coming up with unique names